src/components/chatroom/Chatroom.tsx

```diff
@@ -25,11 +25,12 @@ export const Chatroom = (props: {
     .orderBy('createdAt')
     .limit(25)
     .where("threadId", "==", id)
-  const [messages]: [
+  const [messages,, err]: [
     IMessage[] | undefined,
     boolean,
     firebase.FirebaseError | undefined
   ] = useCollectionData(query, { idField: 'id' })
+  if (err) console.error(err)
```

Comment:
for development purposes this is fine but, i would prefer `err` & logging to console removed from production